api/bazel/repository_locations.bzl

```diff
@@ -14,11 +14,11 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "protoc-gen-validate (PGV)",
         project_desc = "protoc plugin to generate polyglot message validators",
         project_url = "https://github.com/envoyproxy/protoc-gen-validate",
-        version = "278964a8052f96a2f514add0298098f63fb7f47f",
-        sha256 = "e368733c9fb7f8489591ffaf269170d7658cc0cd1ee322b601512b769446d3c8",
+        version = "ec9cd95372b9630aafba2cb4a0e448c43768e0df",
+        sha256 = "60b4b8036497368512795afca0292d1777d1ce651db76c36706af1733179213a",
```

Comment:
Can we bump to the latest commit?
`1bcea29601b5624234a19b3d7f0ebd9e9984f583`
`2062bbe50eddf3c98490339721fb02b5b5cd78f610f163b98bbf95ba7105553f`

Reply:
Done